src/website/views/javascript/tabs.pug

```diff
@@ -350,7 +351,7 @@ section.a-table.a-table__options.-bordered.-my--3
               | Makes the browser wait for the sliding border to move to the clicked link before following links.
               | It only works with common links.
 
-h3#navigationExampleOverflowMenu Overflow Menu
+h3 Overflow Menu
```

Comment:
Removing caused page to jump when links are clicked

Reply:
Actually I figured out that this ID was overwritten by chi-docs.js anchor generator and since this ID didn't exist clicking tabs did nothing. Now the page jumps because tabs have corresponding hash targets in the DOM. So in order to assign to tabs some existing element and avoid jumping behaviour, example tabs need to target some element with fixed position on the top of the page. So I added a -sr--only element with fixed position. What do you think about this approach? @mattnickles @jllr